manageiq-appliance_console.gemspec

```diff
@@ -25,11 +25,13 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "awesome_spawn",           "~> 1.6"
   spec.add_runtime_dependency "bcrypt",                  "~> 3.1.10"
   spec.add_runtime_dependency "bcrypt_pbkdf",            ">= 1.0", "< 2.0"
+  spec.add_runtime_dependency "ed25519",                 ">= 1.2", "< 1.3"
   spec.add_runtime_dependency "highline",                "~> 2.1"
   spec.add_runtime_dependency "i18n",                    ">= 0.8"
   spec.add_runtime_dependency "linux_admin",             "~> 2.0"
   spec.add_runtime_dependency "manageiq-password",       "< 2"
-  spec.add_runtime_dependency "net-scp",                 "~> 1.2.1"
+  spec.add_runtime_dependency "net-scp",                 "~> 4.0"
+  spec.add_runtime_dependency "net-ssh",                 "~>7.2"
```

Comment:
@bdunne Please review - I'm curious if there are follow on effect of this large of a bump in both of these.